test/scripts/v1alpha3/run-tests.sh

```diff
@@ -139,5 +139,6 @@ done
 echo "Running e2e hyperopt random experiment"
 export KUBECONFIG=$HOME/.kube/config
 go run run-e2e-experiment.go ../../../examples/v1alpha3/hyperopt-random-example.yaml
+go run run-e2e-experiment.go ../../../examples/v1alpha3/skopt-bayesian-optimization-example.yaml
```

Comment:
Image has also to be replaced with CI imagee

Reply:
I am not sure what's your meaning. Can you explain more?

Reply:
i meant this - https://github.com/kubeflow/katib/blob/master/test/scripts/v1alpha3/run-tests.sh#L89

Reply:
Oh yes, thanks.